tensorflow/compiler/mlir/tensorflow/ir/tf_generated_ops.td

```diff
@@ -4783,9 +4783,47 @@ select(condition, t, e) ==> [[1, 2],
 }
 
 def TF_SelectV2Op : TF_Op<"SelectV2", [NoSideEffect]> {
-  let summary = "";
+  let summary = "Selects elements from `x` or `y`, depending on `condition`.";
```

Comment:
This is a generated file; don't edit generated files.

I believe this is generated from the api_def file for selectv2

Reply:
I guess I don't really understand the semantics of the ops definitions, particularly the endpoint part. Guess it is not the critical part for now. If you could explain it, I can add it in PR too. Sorry for   my lack of knowledge of the code base.